backend/resources/donations.py

```diff
@@ -97,6 +97,14 @@ def donations_post(session, user_inst):
     if balance < int(amount):  # ToDo: gas-cost?
         return jsonify({'error': 'not enough balance'}), 406
 
+    # check if project has Milestone
+    milestoneresults = session.query(Milestone).filter(Milestone.project_id == idproject)
+    milestones = []
+    for row in milestoneresults:
+        milestones.append(row)
```

Comment:
```suggestion
    milestones = list(session.query(Milestone).filter(Milestone.project_id == idproject))
```

Oder besser noch mit count